pkg/util/ctxgroup/ctxgroup.go

```diff
@@ -138,6 +138,9 @@ type Group struct {
 // returns an error, even if no Go invocation did. In particular, calling
 // Wait() after Done has been closed is guaranteed to return an error.
 func (g Group) Wait() error {
+	if g.ctx == nil {
```

Comment:
what situation resulted in this diff?

Reply:
it lets me unconditionally call grp.Wait in batcher, without needing to test if it was ever initialized